api/src/main/java/org/pmiops/workbench/db/dao/ParticipantCohortAnnotationDao.java

```diff
@@ -4,6 +4,8 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.repository.query.Param;
 
+import java.util.List;
+
 public interface ParticipantCohortAnnotationDao extends JpaRepository<ParticipantCohortAnnotation, Long> {
 
     ParticipantCohortAnnotation findByCohortReviewIdAndCohortAnnotationDefinitionIdAndParticipantId(
```

Comment:
We should probably add a (unique) index on these columns.

Reply:
Done.